app/pages/links/link_new.php

```diff
@@ -0,0 +1,26 @@
+<?php
+restrict_access(Access::$EDIT_USERS);
+$v = new Validator();
+$url = $v->url("url")->placeholder("Url")->required();
+$button_text = $v->text("button_text")->placeholder("Nom")->required();
+$description = $v->textarea("description")->placeholder("Description")->required();
+
+if ($v->valid()) {
+    $link = new Link($url->value, $button_text->value, $description->value);
+    em()->persist($link);
+    em()->flush();
+    redirect(get_header("HX-Current-Url"));
```

Comment:
We might already have a shortcut for this, but I am not sure. Something like `reload()`

Reply:
There isn't actually, and we should add it : this way you can create an htmx "component" with a form validation that redirects to the same page the component is called :-)